src/main/java/org/testcontainers/containers/TarantoolContainer.java

```diff
@@ -18,7 +18,7 @@ public class TarantoolContainer extends GenericContainer<TarantoolContainer>
         implements TarantoolContainerOperations<TarantoolContainer> {
 
     public static final String TARANTOOL_IMAGE = "tarantool/tarantool";
-    public static final String DEFAULT_IMAGE_VERSION = "2.11.2-centos7";
+    public static final String DEFAULT_IMAGE_VERSION = "2.11.2-ubuntu20.04";
```

Comment:
About our prev conversation in total how it's relative to CartridgeContainer?

Reply:
You want no leave default container version?

Reply:
It does not, we can leave the previous value

Reply:
I'v just wanted to use the same container